README.md

```diff
@@ -45,30 +45,38 @@ It is **not production code** and requires code-style or python best practices t
 | [CWE-665: Improper Initialization](CWE-664/CWE-665/.)                                                                                          ||
 | [CWE-833: Deadlock](CWE-664/CWE-833/.)                                                                                                         ||
 | [XXX-005: Consider hash-based integrity verification of byte code files against their source code files](CWE-664/XXX-005/.)                    ||
+|<img width=680>|<img width=140>|
+
+| <div style="width:500px">[CWE-682: Incorrect Calculation](https://cwe.mitre.org/data/definitions/682.html)</div> |<div style="width:120px">Prominent CVE</div>|
```

Comment:
that <div style="width:500px" does not work.
I removed it in the new README.md and only have the <img> in an empty row at the end.
but only remove the div you edited otherwise we get merge conflicts